vtex-apps/minicart
# Add 'customPixelEventId' prop

#### What problem is this solving?

Enables users to configure the `minicart.v2` in such a way that it opens once a certain event with `id === customPixelEventId` is fired somewhere in the `window`. 

This adds support for the `customPixelEventId` prop from the `Drawer` component, used in the `drawer` variation of `minicart.v2`.

#### How to test it?

Go to this [Workspace](https://customevents--storecomponents.myvtex.com/) and add an item into the cart. you should see the minicart opening itself.

#### Screenshots or example usage:

![Aug-26-2020 13-36-52](https://user-images.githubusercontent.com/27777263/91332237-45346000-e7a2-11ea-9f78-091a0ae13cfb.gif)

#### Related to / Depends on

Depends on https://github.com/vtex-apps/drawer/pull/34.

#### How does this PR make you feel? [:link:](http://giphy.com/)



![](https://media.giphy.com/media/BxWTWalKTUAdq/giphy.gif)
